libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/components/ListItem.kt

```diff
@@ -119,7 +119,7 @@ fun ListItem(
 
     androidx.compose.material3.ListItem(
         headlineContent = decoratedHeadlineContent,
-        modifier = Modifier.clickable(enabled = enabled && onClick != null, onClick = onClick ?: {}).then(modifier),
+        modifier = if (onClick != null) Modifier.clickable(enabled = enabled, onClick = onClick).then(modifier) else modifier,
```

Comment:
Not new in this PR, but I am wondering if this should not be 
```kotlin
modifier = modifier.then(if (onClick != null) Modifier.clickable(enabled = enabled, onClick = onClick) else modifier,
```
instead. WDYT?

Reply:
That breaks the ripple effect if we set any padding.